applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.cpp

```diff
@@ -40,14 +63,15 @@ namespace Kratos
     void ShockCapturingPhysicsBasedProcess::ExecuteInitialize()
     {
         // Initialize nodal values
-        block_for_each(mrModelPart.Nodes(), [&](Node<3>& rNode) {
+        block_for_each(mrModelPart.Nodes(), [&](Node<3> &rNode)
+                       {
             if (mShockSensor) {rNode.SetValue(ARTIFICIAL_BULK_VISCOSITY, 0.0);}
             if (mShearSensor) {rNode.SetValue(ARTIFICIAL_DYNAMIC_VISCOSITY, 0.0);}
-            if (mShockSensor || mThermalSensor) {rNode.SetValue(ARTIFICIAL_CONDUCTIVITY, 0.0);}
-        });
+            if (mShockSensor || mThermalSensor) {rNode.SetValue(ARTIFICIAL_CONDUCTIVITY, 0.0);} });
 
         // Initialize elemental values
-        block_for_each(mrModelPart.Elements(), [&](Element& rElement){
+        block_for_each(mrModelPart.Elements(), [&](Element &rElement)
+                       {
```

Comment:
Please revert this kind of aesthetic changes. These blurry the real code modifications.